deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -o errexit
+
+ccnp_url=https://github.com/intel/confidential-cloud-native-primitives.git
+device_plugin=ccnp-device-plugin:0.1
+get_quote=ccnp-quote-server:0.1
+measurement=ccnp_measurement_server:0.1
+eventlog=ccnp_eventlog_server:0.1
```

Comment:
How about making all container images and version can be customized? But a default name can be set in advance.